templates/smartstart/index.html

```diff
@@ -0,0 +1,508 @@
+          alt="Snaps",
+          width="200",
+          height="120",
+          hi_def=True,
+          attrs={"class": "u-hide--small", "style":"height: 120px; width: auto;"},
+        ) | safe
+      }}
+      </div>
+      <h4>App store</h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Private branded IoT app store included</li>
+        <li class="p-list__item is-ticked">Secured, hosted and managed</li>
+        <li class="p-list__item is-ticked">Global CDN for software delivery</li>
+        <li class="p-list__item is-ticked">CI/CD pipeline integration</li>
+      </ul>
+    </div>
+    <div class="col-6">
+    <div>
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/d2387483-telecommunications-grey.svg",
+          alt="",
+          width="100",
+          height="100",
+          hi_def=True,
+          attrs={"class": "u-hide--small", "style":"height: 120px; width: auto;"},
+        ) | safe
+      }}
+      </div>
+      <h4>Software updates</h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Choose suitable update frequency</li>
+        <li class="p-list__item is-ticked">Automatic roll-backs on failure</li>
+        <li class="p-list__item is-ticked">Metered billing</li>
+        <li class="p-list__item is-ticked">Single secure source of updates</li>
+      </ul>
+    </div>
+  </div>
+  <div class="row u-equal-height u-sv3">
+    <div class="col-6 u-sv3">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/dbd97444-partner-with-us.svg",
+          alt="",
+          width="200",
+          height="155",
+          hi_def=True,
+          attrs={"class": "u-hide--small", "style":"height: 120px; width: auto;"},
+        ) | safe
+      }}
+      <h4>Services</h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Knowledge transfer workshops</li>
+        <li class="p-list__item is-ticked">On-demand training content</li>
+        <li class="p-list__item is-ticked">Engineering consulting</li>
+      </ul>
+    </div>
+    <div class="col-6 u-sv3">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/e9d3cf66-We+support+your+operations.svg",
+          alt="",
+          width="100",
+          height="100",
+          hi_def=True,
+          attrs={"class": "u-hide--small", "style":"height: 120px; width: auto;"},
+        ) | safe
+      }}
+      <h4>Support</h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">After-sales support</li>
+        <li class="p-list__item is-ticked">Ticket resolution</li>
+        <li class="p-list__item is-ticked">Long term commercial partnership</li>
+      </ul>
+    </div>
+  </div>
+  <div class="u-fixed-width u-align--center">
+    <div>
+      <p class="u-align--center"><i>“Ubuntu is pervasive in business and has a rich ecosystem of partners to accelerate time to market and time to value for organizations that choose to utilize SMART START.”</i></p>
```

Comment:
Please use the Vanilla quote pattern: https://vanillaframework.io/docs/patterns/pull-quote